packages/apply-body-checksum-middleware/src/index.spec.ts

```diff
@@ -2,14 +2,14 @@ import { applyBodyChecksumMiddleware } from "./";
 import { HashConstructor, HttpRequest } from "@aws-sdk/types";
 
 describe("applyChecksumMiddleware", () => {
-  const mockEncoder = jest.fn(() => "encoded");
-  const mockHashUpdate = jest.fn(() => {});
-  const mockHashDigest = jest.fn(() => new Uint8Array(0));
```

Comment:
Just out of curiosity, is this code allowed in jest 24?

Reply:
`jest.fn().mockReturnValue(value)` is just a simple sugar function for `jest.fn(() => value)`
It's backward compatible, but was removed as it was failing `pretest`